File: Image_Rotation.m
function [] = Image_Rotation(n,A)

Resultant=[];

if n==length(A)
    for i=1:length(A)
        %disp(i)
        for j=1:length(A)
            %disp(j)
            x=(n+1)-j;
            Resultant(i,j)=A(x,i);
        end
    end
end
disp(Resultant)  
end

File: Image_Rotation.py
R=int(input('Rows '))

matrix=[]
for i in range(R):
    a=[]
    for j in range(R):
        a.append(int(input()))
    matrix.append(a)

#print(matrix)

Result=[]
for i in matrix:
    #print(i)
    Resultant=[]
    for j in matrix[i]:
        print(j)
        #x=R-j
        matrix[i][j]=matrix[x][i]
        Resultant.append(matrix[x][i])
    Result.append(Resultant)

print(Result)


